ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/sbom/index.tsx

```diff
@@ -128,7 +132,15 @@ const SBOMComponent = () => {
                 <TooltipTrigger>
                   <img
                     alt='CycloneDX'
-                    src={theme === 'dark' ? CycloneDXLight : CycloneDXDark}
+                    src={
+                      theme === 'dark'
+                        ? CycloneDXLight
+                        : theme === 'light'
+                          ? CycloneDXDark
+                          : systemTheme === 'dark'
+                            ? CycloneDXLight
+                            : CycloneDXDark
```

Comment:
Isn't there a way to do just *one* query about the theme in use (no matter matter whether set explicitly, or by adhering to the system theme), and then only have *one* conditional in here like `activeTheme == 'dark' ? CycloneDXLight : CycloneDXDark`?

Reply:
Looks to me like it should work when [using React's `useTheme`](https://reactnavigation.org/docs/use-theme/) instead of [our own `useTheme` implementation](https://github.com/eclipse-apoapsis/ort-server/blob/52a7d1491575a00110ab628a7dfb7ca2fa6b0c0d/ui/src/components/theme-provider-context.tsx#L30-L37).

Reply:
> Looks to me like it should work when [using React's `useTheme`](https://reactnavigation.org/docs/use-theme/) instead of [our own `useTheme` implementation](https://github.com/eclipse-apoapsis/ort-server/blob/52a7d1491575a00110ab628a7dfb7ca2fa6b0c0d/ui/src/components/theme-provider-context.tsx#L30-L37).

That's not a React built-in, but a hook from a library called React Navigation.